pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -1301,3 +1301,19 @@ def test_group_mean_datetime64_nat(input_data, expected_output):
 
     result = data.groupby([0, 0, 0]).mean()
     tm.assert_series_equal(result, expected)
+
+
+def test_groupby_mean_complex():
+    # GH#43701
```

Comment:
can parmaetirze this test instead. also can you add the failing ones in another test, eg.. test_groupby_op_complex_error (e.g. min, max, etc, e.g the non-working ones)